ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -451,16 +457,19 @@ tests:
     workflow: openshift-e2e-openstack-ipi
 - as: e2e-azurestack
   optional: true
+  skip_if_only_changed: (^(docs|upi|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|images)/(alibabacloud|aws|azure|baremetal|gcp|ibmcloud|libvirt|nutanix|openstack|ovirt|powervs|vsphere)/)
   steps:
     cluster_profile: azurestack
     workflow: openshift-e2e-azurestack
 - as: e2e-azurestack-upi
   optional: true
+  skip_if_only_changed: (^(docs|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|upi/.*|pkg/.*|images)/(alibabacloud|aws|azure||baremetal|gcp|ibmcloud|libvirt|nutanix|openstack|ovirt|powervs|vsphere)/)
```

Comment:
I think we should remove this and make the trigger for UPI jobs more specific (e.g. when changes are made to upi files). I can follow up on this in a subsequent PR.